deployment/config.go

```diff
@@ -19,6 +19,9 @@ import (
 type Config struct {
 	ClusterName      string // Name of the cluster.
 	AppInstanceCount int    // Number of application instances.
+	// Number of agents; at least 2 agents required so that one of them will work
+	// as a coordinator.
+	AgentCount       int
```

Comment:
I was thinking to use the coordinator in one of the machines as agent. A coordinator does not consume any extra resources, so it's fine to use it on the same machine.

Reply:
From a theoretical point of view I would prefer the coordinator to be separated from the agents it controls.  

A server running an agent with many controllers/users might indeed experience a lot of pressure. The coordinator completes the feedback loop and it's important that it can act as fast as possible when data comes in from prometheus.

That said, it's probably fine for now to have a shared instance (load-agent and coordinator) and see how that works.

The coordinator could also be run locally by the user doing the load-test since it's the initiator.

Reply:
Hmm .. fine by me.

Reply:
We are going with same machine for now. Slating this for a later improvement.